### ISSUE TITLE
Adicionar Flask-Profiler

### ISSUE THREAD

**Issue by rochacbruno** [rochacbruno/talkshow#7]
Adicionar um ext/profiling.py para incluir Flask-Profiler

**Comment by neiesc** [comment]
@rochacbruno seria esse o Flask-Profiler https://github.com/muatik/flask-profiler ?

**Comment by rochacbruno** [comment]
Sim! a ideia é criar uma configuração aqui: https://github.com/rochacbruno/talkshow/tree/extended/talkshow/ext e então colocar o decorator nas views, e deixar o profiler opcional via variavel de ambiente assim como no debug.py
